e2e/tests/components/menu/menu.e2e-spec.ts

```diff
@@ -0,0 +1,46 @@
+import { browser } from 'protractor';
+import { imageCompare } from '../common/image-compare';
+import { MenuPage } from './menu.po.spec';
+
+describe('Menu', () => {
+
+    let page: MenuPage;
+
+    beforeEach(async () => {
+        page = new MenuPage();
+        await page.getPage();
+    });
+
+    it('should change menu placement after initialization', async () => {
+
+        // open menu and expect placement to be to the left
```

Comment:
Instead of comments, just make each of these individual tests, then the description becomes part of the test and you can see exactly what has failed.